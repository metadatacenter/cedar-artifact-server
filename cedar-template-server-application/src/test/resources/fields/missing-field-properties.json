{
    "@type": "https://schema.metadatacenter.org/core/TemplateField",
    "@context": {
        "xsd": "http://www.w3.org/2001/XMLSchema#",
        "pav": "http://purl.org/pav/",
        "oslc": "http://open-services.net/ns/core#",
        "schema": "http://schema.org/",
        "pav:createdOn": {
            "@type": "xsd:dateTime"
        },
        "pav:createdBy": {
            "@type": "@id"
        },
        "pav:lastUpdatedOn": {
            "@type": "xsd:dateTime"
        },
        "oslc:modifiedBy": {
            "@type": "@id"
        }
    },
    "type": "object",
    "title": "A sample text field schema",
    "description": "A sample text field schema autogenerated by the CEDAR Template Editor 1.0.3",
    "_ui": {
        "title": "A sample text field",
        "description": "",
        "inputType": "textfield"
    },
    "_valueConstraints": {
        "requiredValue": false
    },
    "required": [
        "@value"
    ],
    "pav:createdOn": "2017-03-28T16:19:14-0700",
    "pav:createdBy": "https://metadatacenter.org/users/74c2d307-af38-43a3-9ca2-5c644d2a4e58",
    "pav:lastUpdatedOn": "2017-03-28T16:19:14-0700",
    "oslc:modifiedBy": "https://metadatacenter.org/users/74c2d307-af38-43a3-9ca2-5c644d2a4e58",
    "additionalProperties": false,
    "@id": "https://repo.staging.metadatacenter.net/template-fields/fc9e8c76-4249-4acf-a0f9-91e14bb734b0",
    "$schema": "http://json-schema.org/draft-04/schema#"
}