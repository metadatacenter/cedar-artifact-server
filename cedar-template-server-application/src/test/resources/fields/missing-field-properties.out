{"validates":"false","warnings":[],"errors":["No JSON Schema properties field in artifact at path /"]}